Added rambo community model that was missing

kiariendegwa opened the pull request:
Hi team, 

Thank you, soo much for working on this model and detailing everything so meticulously.
I, however, found that you are actually missing the Rambo model as described in the README diagram.
I, therefore, implemented the model, using Keras Sequential modelling so as to keep it consistent with the rest of your code, hope you merge this PR just for future clarity's sake.

Keep up the good work!

Cheers, 
Kiarie

Commit:
Added rambo community model that was missing

mvirgo reviewed (changes requested):
Can you remove the changes to the files other than rambo/train.py, and remove the duplication of "merge, Merge" in train.py? Otherwise I can merge the PR.

Commit:
Fix: remove double import of Merge

Commit:
Delete solution-komanda-checkpoint.ipynb

Commit:
fix: revert Komanda notebook to Python 2

Code is written in Python 2 (take note of print statements), so reverting the underlying notebook versions to Python 2.

Commit:
fix: one more Python 2 fix

mvirgo commented:
Made the changes in my above commits; merging. Thanks for adding this!
